sdk/digitaltwins/Azure.DigitalTwins.Core/tests/prerequisites/prerequisite readme.md

```diff
@@ -13,13 +13,19 @@
   - If it isn't, update it
 - Use this link to install [Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest])
 
-## Delete
+### Install Bicep
 
-To delete the digital twins instance, you need to first delete the endpoint added by the script (the service doesn't yet support cascading delete).
+- Install using the instructions in [bicep](https://github.com/Azure/bicep/blob/main/docs/installing.md)
 
-1. To do this, run the command `az dt endpoint delete -n <dt name> -g <rg name> --en someEventHubEndpoint`.
-1. If you have other endpoints that have been added outside this script, you can discover them with the command `az dt endpoint list -n <dt name> -g <rg name>`.
-1. Then delete them with the same command in step 1.
+### Run the setup script - The script outputs [test-resources.bicep](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/digitaltwins/test-resources.bicep) file in the root folder [digitaltwins](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/digitaltwins)
+
+- `.\setup.ps1`
+
+### Generate the ARM template [test-resources.json](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/digitaltwins/test-resources.json) from the [test-resources.bicep](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/digitaltwins/test-resources.bicep) file using the command
+
+- `bicep build ..\..\..\test-resources.bicep`
```

Comment:
Can we make a helper script to run this command line? Especially with the paths down it is getting more complicated.

Reply:
The scrip will just have 1 line. Do you think it is necessary? I don't mind adding one.

Reply:
I wouldn't have asked if I didn't think it was necessary. I did the same thing in track 2 for things like exporting APIs and updating snippets.